frontend/src/bundles/profile/components/profile-settings/profile-settings.tsx

```diff
@@ -34,10 +39,16 @@ type Properties = {
 };
 
 const ProfileSettings: React.FC<Properties> = ({ onSubmit, isLoading }) => {
+    const dispatch = useAppDispatch();
+
     const [valuesDefault, setValuesDefault] = useState(false);
-    const { user } = useAppSelector(({ auth }) => ({
-        user: auth.user,
-    }));
+    const { user, userBonusesTransactions, userBonusesStatus } = useAppSelector(
+        ({ auth }) => ({
+            user: auth.user,
+            userBonusesTransactions: auth.userBonusesTransactions,
+            userBonusesStatus: auth.userBonusesStatus,
+        }),
+    );
```

Comment:
just select whole slice and use destructuring. this approach causes warnings as you creating new object instance on every select